packages/tools/stylelint-no-logical-props-shorthands/__tests__/plugin.test.js

```diff
@@ -68,7 +68,10 @@ describe('stylelint-no-logical-props-shorthands', () => {
                         fix: true,
                     });
 
-                    expect(result.output).toMatchSnapshot();
+                    expect(result.code).toMatchInlineSnapshot(`
```

Comment:
Very nice. I'd be tempted to do `result.code.trim()` but not sure if that's in the spirit of things.

Reply:
This change was a mistake actually, reverted.